extensions/positron-python/python_files/positron_requirements/requirements.in

```diff
@@ -0,0 +1,16 @@
+# This file is used to generate requirements.txt.
+# To update requirements.txt, run the following commands.
+# Use Python 3.8 when creating the environment or using pip-tools
```

Comment:
Should we change this to 3.9 (or just wait until we officially drop 3.8)?

Reply:
Will do this when we drop 3.9